<launch>
    <!-- Gazebo launch -->
    <include file="$(find mrob_gazebo)/launch/mrob_gazebo.launch">
        <arg name="world_file" value="$(find mrob_highlevel_controller)/worlds/singlePillar.world"/>
        <arg name="enable_ekf" value="true"/>
    </include>

    <!-- Controller launch -->
    <node name="HighlevelController" pkg="mrob_highlevel_controller" type="controller" output="screen">
        <rosparam command="load" file="$(find mrob_highlevel_controller)/config/params.yaml"/>
    </node>

    <!-- Rviz launch -->
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find mrob_highlevel_controller)/rviz/config.rviz"/>
</launch>